### PR TITLE
[2.26.x] Support default sort parameters for WFS 1.1.0 sources.

### PR DESCRIPTION
**Pull request by glenhein**
#### What does this PR do?
Support default sort parameters for WFS 1.1.0 sources.

#### Who is reviewing it? 
@derekwilhelm 
@jlcsmith 

#### How should this be tested?
Configure a WFS source that does not support RELEVANCE. Leave the default sort property and default sort order blank in the config. Confirm that a search that sorts by RELEVANCE returns unsorted results. Change the config to use "name" and "ASCENDING" for the default property and order. Confirm that a search that sorts by RELEVANCE returns results sorted by name.

#### What are the relevant tickets?
Fixes: #____

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Threat Dragon models
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Review by jlcsmith — APPROVED** [review]

**Review by alexabird — APPROVED** [review]

**Comment by alexabird** [comment]
build now

File: catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-common/src/main/java/org/codice/ddf/spatial/ogc/wfs/v110/catalog/common/Wfs11Constants.java [hunk 1]
```diff
@@ -55,6 +55,13 @@ public class Wfs11Constants extends WfsConstants {
 
   public static final QName MULTI_POLYGON = new QName(GML_3_1_1_NAMESPACE, "MultiPolygon");
 
+  /**
+   * This is a list of sort-by attribute names that were removed from the query because they are
+   * known to be unsupported by the source. In practice, this value will be appended with
```
Comment (jrnorth) [inline catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-common/src/main/java/org/codice/ddf/spatial/ogc/wfs/v110/catalog/common/Wfs11Constants.java:60]: ✏️ ~appended~ suffixed

File: catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/src/test/java/org/codice/ddf/spatial/ogc/wfs/v110/catalog/source/WfsSourceTest.java [hunk 2]
```diff
@@ -1670,75 +1671,75 @@ public void testSortingDisabled() throws Exception {
...
     setUpMocks(null, null, ONE_FEATURE, ONE_FEATURE);
     final QueryImpl propertyIsLikeQuery =
         new QueryImpl(builder.attribute(Metacard.ANY_TEXT).is().like().text("literal"));
     setupMapper(
         MOCK_TEMPORAL_SORT_PROPERTY, MOCK_RELEVANCE_SORT_PROPERTY, MOCK_DISTANCE_SORT_PROPERTY);
     source.setMetacardMappers(metacardMappers);
-    propertyIsLikeQuery.setSortBy(new SortByImpl(new PropertyNameImpl(null), null));
+    propertyIsLikeQuery.setSortBy(new SortByImpl(Result.TEMPORAL, "foo"));
+
+    source.setDefaultSortName(Result.TEMPORAL);
```
Comment (jrnorth) [inline catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/src/test/java/org/codice/ddf/spatial/ogc/wfs/v110/catalog/source/WfsSourceTest.java:1731]: ❓Should we check that the query ultimately falls back to the default?

**Comment by glenhein** [comment]
build now

File: catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/src/test/java/org/codice/ddf/spatial/ogc/wfs/v110/catalog/source/WfsSourceTest.java [hunk 3]
```diff
@@ -1670,75 +1672,111 @@ public void testSortingDisabled() throws Exception {
...
     source.setMetacardMappers(metacardMappers);
-    propertyIsLikeQuery.setSortBy(new SortByImpl(null, "ASC"));
+    propertyIsLikeQuery.setSortBy(new SortByImpl(Result.TEMPORAL, "foo"));
 
-    source.query(new QueryRequestImpl(propertyIsLikeQuery));
+    ArgumentCaptor<ExtendedGetFeatureType> argumentCaptor =
+        ArgumentCaptor.forClass(ExtendedGetFeatureType.class);
+
+    SourceResponse sourceResponse = source.query(new QueryRequestImpl(propertyIsLikeQuery));
+    assertThat(
```
Comment (jrnorth) [inline catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/src/test/java/org/codice/ddf/spatial/ogc/wfs/v110/catalog/source/WfsSourceTest.java:1727]: ✏️ You don't need to change this, but I will point out that Hamcrest has a `hasEntry()` matcher: https://hamcrest.org/JavaHamcrest/javadoc/1.3/org/hamcrest/collection/IsMapContaining.html

**Review by jrnorth — APPROVED** [review]

**Comment by alexabird** [comment]
✅ Hero looks good!

**Pull request merged**